spec/change_set_persisters/change_set_persister_spec.rb

```diff
@@ -1426,6 +1426,84 @@
         expect(File.exist?(disk_preservation_path.join(resource.id.to_s, "#{resource.id}.json"))).to eq false
       end
     end
+
+    context "when deleting a numismatic place resource" do
+      it "assigns a title to the deletion marker" do
+        resource = FactoryBot.create_for_repository(:numismatic_place)
+        change_set_persister.delete(change_set: ChangeSet.for(resource))
+
+        deletion_markers = change_set_persister.query_service.find_all_of_model(model: DeletionMarker)
+        deletion_marker = deletion_markers.find { |m| m.resource_id == resource.id }
+        expect(deletion_marker.resource_title).not_to be_nil
```

Comment:
Is this title dynamically generated? I think it would be nice if we could test for the string we expect, rather than not nil.

Reply:
They are generated. Added.